packages/mars-theme/src/components/twitt-tooltip/twitt-icon.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function TwittIcon({ size, color }) {
+  const styles = {
+    svg: {
+      fill: '#000000',
+    },
+    g: {
+      mixBlendNode: 'normal',
+    },
+  };
+
+  return (
+    <svg
```

Comment:
I would prefer if we imported SVGs directly in the components through webpack. You can set the import to an image tag `<img src={twitterIcon} />` for example.